scripts/make_documentation.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -e
+set -x
+
+SCRIPT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
```

Comment:
space around parenthesis is not necessary. User `pwd -P` to get the physical location to avoid chained symlink, i.e., a -> b -> c and `c` is where source scripts are stored.